pandas/plotting/_core.py

```diff
@@ -2704,18 +2704,56 @@ def __call__(self, x=None, y=None, kind='line', ax=None,
 
     def line(self, x=None, y=None, **kwds):
         """
-        Line plot
+        Plot `DataFrame` columns as lines.
+
+        This function is useful to plot lines using `DataFrame`'s values
+        as coordinates.
+        It allows to specify which columns to use for the axes.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : int, str, list of int or list of str, optional
+            It defines the X axis coordinates.
```

Comment:
Remove "it". Just have "Column to use for the horizontal axis." and then the rest of your description.